### PR TITLE
Delete docs/Tutorials/HelloWorld directory

### PR DESCRIPTION
**Pull request by hAkselS**
The Hello World tutorial has been moved to [fprime-community/fprime-tutorial-hello-world](https://github.com/fprime-community/fprime-tutorial-hello-world), hence it needs to be removed from here.

### CODE REVIEW

File: docs/Tutorials/README.md [hunk 1]
```diff
@@ -8,27 +8,22 @@ users to learn F´ and walk through the most basic steps in developing an F´ ap
 
 ## Tutorials Index
 
-1. [HelloWorld](HelloWorld/Tutorial.md)
+1. [HelloWorld](https://github.com/fprime-community/fprime-tutorial-hello-world.git)
```
Comment (LeStarch) [inline docs/Tutorials/README.md:11]: Do you want to link to the published branded page, or directly the repository?  Same applies to all these links.

**Comment by LeStarch** [comment]
@hAkselS lookd like you did not base your work off `devel` but rather off `master` can you correct?

**Comment by thomas-bc** [comment]
I wonder, would it be a good idea to instead of deleting the whole thing, leaving an unreferenced `README.md` in `docs/HelloWorld/README.md` that says something like 
```markdown
HelloWorld tutorial has been reworked and moved to [fprime-community/fprime-tutorial-hello-world](https://github.com/fprime-community/fprime-tutorial-hello-world)
```

I suspect some users are browsing tutorials through GitHub and it would probably throw them off if the entire folders just disappeared?

**Comment by thomas-bc** [comment]
Closing in favor of #2095

**Pull request closed without merging**